LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2021 LootLocker
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "LootLockerServerResponse.h"
+
+#include "LootLockerServerLeaderboardArchiveRequestHandler.generated.h"
+
+USTRUCT(BlueprintType)
+struct FLootLockerServerLeaderboardArchivePlayer
+{
+	GENERATED_BODY()
+	// The name of the Player.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString name;
+	// The Public UID of the Player.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString public_uid;
+	// The ID of the Player.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	int32 id = 0;
+	// The ULID of the Player.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString player_ulid;
+};
+
+USTRUCT(BlueprintType)
+struct FLootLockerServerLeaderboardArchiveDetails 
+{
+	GENERATED_BODY()
+	// The Player on the archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FLootLockerServerLeaderboardArchivePlayer player;
+	// Metadata if any was supplied.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString metadata;
+	// The Player's member ID on the Archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString member_id;
+	// The Player's rank on the archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	int32 rank = 0;
+	// The Player's Score on the archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	int score = 0;
```

Comment:
Intentional that this an in ant not an int32?

Reply:
Well, no. Or rather, the int32 should be just int :P

Reply:
Fixed